src/modules/storage/LedgerStorage.ts

```diff
@@ -3442,15 +3442,44 @@ export class LedgerStorage extends Storages {
      */
     public getLatestTransactions(limit: number, page: number): Promise<any[]> {
         const sql = `SELECT
-                T.block_height, T.tx_hash, T.tx_fee, T.tx_size, T.type,
-                Sum(IFNULL(O.amount,0)) as amount, B.time_stamp, count(*) OVER() AS full_count
-             FROM
-                 tx_outputs O
-                 INNER JOIN transactions T ON (T.tx_hash = O.tx_hash)
-                 INNER JOIN blocks B ON (B.height = T.block_height)
-             GROUP BY O.tx_hash
-             ORDER BY T.block_height DESC
-             LIMIT ? OFFSET ?;`;
+                    block_height,
+                    tx_hash,
+                    time_stamp,
+                    tx_fee,
+                    tx_size,
+                    type,
+                    amount,
+                    count(*) OVER() AS full_count
+                FROM(
+                    SELECT
+                        (SELECT IFNULL(MAX(height), 0) AS height FROM blocks) AS block_height,
+	                    T.tx_hash,
+                        T.time as time_stamp,
+	                    T.tx_fee,
+                        T.tx_size,
+                        T.type,
+	                    Sum(IFNULL(O.amount,0)) as amount
+	                FROM
+	                    tx_outputs O
```

Comment:
```suggestion
	                    tx_output_pool O
```

Reply:
we can not use tx_output_pool. actully we need 
tx_outputs S
                INNER JOIN tx_input_pool I ON (I.utxo = S.utxo_key)